app/views/layouts/application.html.erb

```diff
@@ -8,11 +8,9 @@
   </head>
   <body>
     <div class="container">
-      <div>
-        <%= link_to "Add Quote", new_quote_path %>
-        <%= link_to "All Quotes", quotes_path %>
+      <div class="navbar-container">
         <% if current_user %>
-          <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
+          <%= render "layouts/navbar" %>
```

Comment:
That's a good point. Could make a second "logged_out_navbar" that is just the brand or something along those lines. Or just show the entire thing regardless but then the ux would be weird to be able to click at different things and they just all send you back to the login page.